Bottom: 594311ce9fb55fdd56005649a005247c28087294
Top:    c173e1c7f150a7cb5f5cdf8f7a39b31c79cedc2e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-06-10 02:23:16 +0100

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 9c6d8ee..d52515f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1238,6 +1238,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1260,6 +1261,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
